### ISSUE TITLE
Avaliar resultados do modelo escolhido

### ISSUE THREAD

**Issue by nando3d3** [fga-eps-mds/2022-2-IsItKbs#55]
## Descrição
Após a escolha do modelo a ser usado para o protótipo da release I, é preciso avaliar os resultados deste modelo.
Existem diversas métricas que podem ser utilizadas para esse fim. [Nesse site](https://www.mariofilho.com/as-metricas-mais-populares-para-avaliar-modelos-de-machine-learning/) é possível encontrar alguns exemplos.

## Tarefas
- [x] Avaliar resultado do modelo escolhido por meio de, no mínimo, duas métricas

## Critério de Aceitação
- [x] Explicar o porquê os resultados do modelo estão ou não aceitáveis

**Comment by G16C** [comment]
### Avaliação
Bom, avaliei o modelo como aceitável nesse primeiro momento. Avaliei usando 3 métricas, além da métrica padrão"acuracy_score". 

Agora uma breve explicação do funcionamento das métricas usadas para avaliar o modelo:

- balanced_acuracy_score: A _balanced_acuracy_score_ é usada em problemas de classificação binária e multi-class para lidar com conjuntos de dados desequilibrados. É definida como a média do _recall_ obtida em cada classe.

  O melhor valor é 1 e o pior valor é 0 quando ajustado=False.

- f1_score: O _f1_score_ pode ser interpretado como uma média harmônica de _precision_ e _recall_ ,onde um _f1_score_ atinge seu melhor valor em 1 e pior escore em 0. A contribuição relativa de _precision_ e _recall_ para o _f1_score_ é igual. A fórmula para a pontuação da f1 é:

  f1 = 2 * (precision * recall) / (precision + recall)
  No caso multi-class e multi-label, esta é a média da pontuação f1 de cada classe com ponderação dependendo do parâmetro 
  médio.

- recall_score: O _recall_ é a razão tp / (tp + fn) onde tp é o número de verdadeiros positivos e fn o número de falsos negativos. 
  O _recall_ é intuitivamente a capacidade do classificador de encontrar todas as amostras positivas.

  O melhor valor é 1 e o pior valor é 0.

Como para todas as métricas a acurácia foi maior que 80% julguei esse modelo como válido, resalto ainda que todas as métricas usadas tem compatibilidade com casos de classificação binária. Recomendo, antes de classificarem algum modelo ler o manual de métricas do sklearn, para verificar esse a métrica será válida ou não.

Link do manual: https://scikit-learn.org/stable/modules/model_evaluation.html?adlt=strict&toWww=1&redig=48A63F7AF3F746CEBDE63D669AEF580C

**Comment by G16C** [comment]
Como já avaliei o modelo estou fechando essa issue. O modelo com os testes que fiz está na branch features_arthurm, que deverá receber um merge com a devel.
